packages/react-components/components/label/stories/standard/standard.stories.mdx

```diff
@@ -51,10 +51,10 @@ A label can include a button.
 
 <Preview>
     <Story name="button">
-        <Label>
+        <Label button>
```

Comment:
Should we take a similar approach to the icons and accept a Button component here? I am not sure how usage will be if we do this.

There is a few advantages though:

- We could enforce the button type e.g. ghost
- We could enforce the button size e.g. tiny

While writing this I am wondering if maybe instead we should only accept the property `buttonIcon` and enforce everything else?

Reply:
Extracted to: https://github.com/gsoft-inc/sg-orbit/issues/158